lib/mayaUsd/fileio/translators/translatorCamera.cpp

```diff
@@ -470,9 +471,17 @@ bool _ReadToCamera(
     }
 
     usdAttr = usdCamera.GetFStopAttr();
-    plugName = _tokens->MayaCameraAttrNameFStop;
-    if (!_TranslateUsdAttributeToPlug(usdAttr, cameraFn, plugName, args, context)) {
-        return false;
+    if (usdAttr.IsAuthored()) {
+        plugName = _tokens->MayaCameraAttrNameFStop;
+        if (!_TranslateUsdAttributeToPlug(usdAttr, cameraFn, plugName, args, context)) {
+            return false;
+        }
+        // Enable depthOfField
+        plugName = _tokens->MayaCameraAttrNameDepthOfField;
+        MPlug plug = cameraFn.findPlug(plugName.GetText(), true, &status);
```

Comment:
I think we should have an else clause which sets Maya's depth of field attribute false when fStop is zero.

Reply:
Can the import/export re-attach to existing cameras ? Otherwise dof is off by default in Maya.